sonar-scanner-engine/src/main/java/org/sonar/batch/mediumtest/TaskResult.java

```diff
@@ -258,15 +258,15 @@ private static TextRange toRange(InputFile file, BatchReport.TextRange reportRan
     return null;
   }
 
-  public BatchReport.Test testExecutionFor(InputFile testFile, String testName) {
+  public ScannerReport.Test testExecutionFor(InputFile testFile, String testName) {
     int ref = reportComponents.get(((DefaultInputFile) testFile).key()).getRef();
     try (InputStream inputStream = FileUtils.openInputStream(getReportReader().readTests(ref))) {
-      BatchReport.Test test = BatchReport.Test.PARSER.parseDelimitedFrom(inputStream);
+      ScannerReport.Test test = ScannerReport.Test.PARSER.parseDelimitedFrom(inputStream);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "PARSER"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)